convert2rhel/main.py

```diff
@@ -183,6 +182,21 @@ def main():
     return 0
 
 
+#
+# Boilerplate Task
+#
+
+
```

Comment:
Not sure if we need this comment here, I would remove them as it is pretty clear that the below function is related to a boilerplate

```suggestion

```

Reply:
These comments are here to group the following functions into sections.  I don't think we can achieve that any other way (except to put them into classes... but I' not a fan of unnecessary classes).

I think I'll leave them in.

Reply:
Looking at them inside of the file, though, it looks like the headings aren't complete so the reason for them is unclear.  I'll add more in the next commit (where I'll also add docstrings)